nemo_text_processing/text_normalization/sv/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
```

Comment:
for this and other new files, please check that year is 2023; there are a few 2021/2022 instances

Reply:
Copyright exists from the moment of creation. The files I copied were copyrighted on those dates, and as I am not the author of them, I have to reproduce as faithfully as possible the copyright notices of those files.

Reply:
Marking this as resolved with a terse answer wasn't exactly the nicest way about this, and I'm sorry about that.

If these files were entirely new, then a copyright notice of 2023 at the top would make sense, but none of them are. They are all derivatives of existing files. For the most part, my modifications are not eligible for copyright protection, so no new copyright was created. In the others, I added my copyright notice below the copyright notice of the original file.

Copyright notices aren't really meaningful any more (it used to be that in the US, if you left them out, you lost copyright), but there are still penalties for fraudulently removing or modifying them: see [here](https://www.copyright.gov/title17/92chap5.html#506). I'm not a lawyer, I'm not even going to try to guess what might or might not be construed as 'fraudulent', so for my part, preserving the copyright notices of the originals as exactly as I got them is the safest bet.

If having this sort of notice is the policy, fine, but I would recommend that you check with your legal representative first (specifically mentioning the word 'derivative'), as this is the sort of thing that could potentially cause them problems if an issue arises.

Reply:
The conditions outlined in the incoming contributing doc are fulfilled, this is resolved.